README.rst

```diff
@@ -14,6 +14,15 @@ After Copying The Template
 --------------------------
 
 You'll want to do a find-and-replace to replace all instances of ``frontend-template-application`` with the name of your new repository.
+You'll also want to update the default port number, ``8080``.
+
+::
+
+    export APP_NAME=my-app-name
+    export APP_PORT=8080
+    grep -rl frontend-template-application . | xargs sed -i.bak "s/frontend-template-application/frontend-app-${APP_NAME}/g"
+    grep -rl 8080 . | xargs sed -i.bak "s/8080/${APP_PORT}/g"
```

Comment:
@davidjoy We may not want to do/document exactly like this, but this is basically what I used locally.

Reply:
I dunno, I feel like we're documenting how to do a find and replace... I think my general thought was that folks can do that part however they're used to; I just used my IDE.  Happy to be convinced otherwise, but that was my gut reaction.  

FWIW, most of the occurrences of frontend-template-application occur in this README.  I wonder if we should just document a list of the 5-6 spots that need to be changed - along with updates to the openedx.yml, for example, and suggest they delete most of the README's contents?

Reply:
I've removed the docs changes for now and will re-open a dialogue on a new PR :)